waffle_utils/run.py

```diff
@@ -24,6 +25,15 @@
 # export docs
 format_docs = f"[{', '.join(map(lambda x: x.name, Format))}]"
 
+# video processing docs
+input_video_path_docs = "input video file path"
+input_images_path_docs = "input image files path"
+output_images_path_docs = "output images path"
+output_video_path_docs = "output video file path"
```

Comment:
I'd like to suggest some naming convention.
`directory` can stand for `directory`.
`path` can stand for `directory` and `file path`.

To elliminate these confusions, I think we should use `directory` only for `directory` and `path` only for `file path`.
@snuailab/waffle. Give me any suggestion.

Reply:
> To elliminate these confusions, I think we should use `directory` only for `directory` and `path` only for `file path`. 
I agree with you.